retis/src/core/events/bpf/include/events.h

```diff
@@ -35,15 +41,22 @@ enum retis_event_owners {
 
 struct retis_raw_event {
 	u16 size;
+	u8 pad[6];
 	u8 data[RAW_EVENT_DATA_SIZE];
-} __attribute__((packed));
+} __packed;
+_Static_assert(__builtin_offsetof(struct retis_raw_event, data) == 8,
+	       "member data must start at offset 8.");
 
 /* Please keep synced with its Rust counterpart. */
 struct retis_raw_event_section_header {
 	u8 owner;
 	u8 data_type;
 	u16 size;
-} __attribute__((packed));
+	u8 pad_size;
+	u8 pad[3];
+} __binding;
```

Comment:
Why not also converting the other event structs? This makes things to be in between 2 states which is not great for maintenance.

Reply:
admittedly, yes, I had the same thought, but this PR has the potential to become huge (more than it is already), that's why I left it out to be converted in a follow-up PR.  Let's just convert RawEvent as well.

Reply:
This should also help in not having to explicitly populate the `pad` fields in the Rust side.